gcloud-java-dns/src/test/java/com/google/gcloud/dns/DnsBatchTest.java

```diff
@@ -528,6 +612,10 @@ public void testApplyChangeRequest() {
 
   @Test
   public void testApplyChangeRequestWithOptions() {
+    EasyMock.verify(dns);
+    EasyMock.reset(dns);
+    EasyMock.expect(dns.options()).andReturn(optionsMock);
+    EasyMock.replay(dns);
```

Comment:
Also here, we are missing to check that the callback correctly forwards the exception. We should probably test this in `testApplyChangeRequest` by using `onFailure` on the callback rather than `error` on the batch result.